.mailmap

```diff
@@ -4,3 +4,6 @@ José Lorenzo Rodríguez <jose.zap@gmail.com>
 José Lorenzo Rodríguez <jose.zap@gmail.com> <lorenzo@users.noreply.github.com>
 José Lorenzo Rodríguez <jose.zap@gmail.com> José Lorenzo Rodríguez Urdaneta <lorenzo@Venus-2.local>
 ADmad <admad.coder@gmail.com> <ADmad@users.noreply.github.com>
+thinkingmedia <mathew@thinkingmedia.ca>
+Thinking Media <thinkingmedia@users.noreply.github.com>
+Mathew Foscarini <mathew@thinkingmedia.ca>
```

Comment:
I think the goal of mailmap is to make the names consistent across all your mail aliases.